### ISSUE TITLE
Remove dependency to Microsoft.AspNetCore.Mvc.Core

### ISSUE THREAD

**Issue by WolfspiritM** [thomasduft/openiddict-ui#22]
There are a few dependencies to Microsoft.AspNetCore.Mvc.Core which seems to not be needed (at least I was able to build with them removed) and it's causing issues when trying to build a self-contained executable cause it seems to have old and outdated dependencies itself.

```
tomware.OpenIddict.UI.Identity.Api 1.7.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.5 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.Runtime.InteropServices (>= 4.3.0)  [/src/Solution.sln]
tomware.OpenIddict.UI.Identity.Api 1.7.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.5 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.Runtime.InteropServices (>= 4.1.0) [/src/Solution.sln]
```

Can this reference be removed?

As a work around I got it working by adding this to my csproj:

```<PackageReference Include="Microsoft.Extensions.DependencyModel" Version="7.0.0" />```

**Comment by thomasduft** [comment]
Hi @WolfspiritM

Sure, will try and remove the mentioned dependency in the very near-term future (maybe til this weekend ;-)).

Thanks for the hint, cheers Thomas

**Comment by thomasduft** [comment]
Hi @WolfspiritM 

Fixed it with v1.9.0. Hope it helps.

Have a nice day, Thomas
